### DISCUSSION TITLE
Avalonia Xaml editor doesn't "Go to Definition"

### DISCUSSION THREAD

**Discussion by cuikp** [AvaloniaUI/AvaloniaVS#419] [General]
In the Xaml Editor (Avalonia 11.0.4, Visual Studio 2022), it seems that the "Go to Definition" context menu item (default Shift-F2) has no effect for me in Avalonia as it does in WPF.   
Is this working for anyone else?

**Comment by JGKle** [comment]
Doesn't work for me either. And in fact, "Go To Definition" doesn't even work for me in .cs, when jumping to Avalonia types (as it does for other types). For example, if I try to "Go To Definition" of `Avalonia.Media.Imaging.WriteableBitmap`, nothing happens. But any other types - i.e. `System.Drawing.Bitmap`, `SkBitmap`, etc work (it jumps to the decompiled definitions).

Can you jump to Avalonia definitions in .cs code...? I definitely have the extension installed & working, as all the project templates are there.

**Reply by cuikp** [reply]
If the definition is within a .cs file, I can jump to the declaration line with Shift-F2 (Go to Definition), but not from Xaml.  (Unless I'm misunderstanding what you mean).

**Reply by JGKle** [reply]
Ok nevermind, fixed. I uninstalled & reinstalled the nuget package, & now it works. Fluke I guess :)

Doesn't work at all in axaml though, as you said.
